kubernetes/cray-opa/templates/policies/keycloak-admin.yaml

```diff
@@ -210,9 +210,10 @@ data:
         {"method": "GET",  "path": `^/apis/bos/v2/version$`}, # GETs allow views on specific BOS version information
         # CFS
         {"method": "GET", "path": `^/apis/cfs/v3/configurations$`}, # GET allows listing one or all all configurations for this tenant
```

Comment:
Do we need both  GET for`^/apis/cfs/v3/configurations$` and `^/apis/cfs/v3/configurations/.*$`?

Reply:
The configurations endpoint will list all of the configurations$, and the configurations/.*$ will let you specify a single configuration.

Reply:
Good enough an answer for me. I just want to make sure we know why for every endpoint we expose.

Reply:
I also updated the comments to better reflect what I just explained to you.